charts/datadog/templates/rbac.yaml

```diff
@@ -109,6 +109,17 @@ rules:
   resources: ["secrets"]
   verbs: ["get"]
 {{- end }}
+{{- if .Values.datadog.helmCheck.enabled}}
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
+{{- end }}
```

Comment:
I think this is more for the "clusterCheckRunner"

Reply:
Yes. Same as this comment: https://github.com/DataDog/helm-charts/pull/548#discussion_r811103112